logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/MLSConversationRepository.kt

```diff
@@ -212,6 +213,11 @@ class MLSConversationDataSource(
             }
         }
 
+    override suspend fun leaveGroup(groupID: String) =
```

Comment:
Maybe we should call it `deleteGroup` to avoid confusion about this creating a leave proposal or something.